containers/rancher-cloud/app/run

```diff
@@ -0,0 +1,146 @@
+#!/bin/sh
+set -xe
+
+action=${CNAB_ACTION:-install}
+name=${CNAB_INSTALLATION_NAME:-rancher-cloud}
+rancher_hostname=${RANCHER_HOSTNAME:-rancher-cloud.my.org}
+rancher_server_url=${RANCHER_SERVER_URL:-https://rancher-cloud.my.org}
+rancher_replicas=${RANCHER_REPLICAS:-1}
+rancher_bootstrap_password=${RANCHER_BOOTSTRAP_PASSWORD:-rancher}
+
+helm_timeout=20m
+
+rancher_image_arguments=
```

Comment:
I don't think we need to override pull policy here since we really don't have a good use case for it. For dev and testing, we can already patch the deployment if needed.